drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -110,6 +113,13 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
         default="",
     )
 
+    max_items_per_zone = Integer(
```

Comment:
It feels odd that there is a single value for all zones in the problem. One can imagine that some zones are much larger than other zones. But product has decided that this is a valuable option to have?